dev/AnimatedVisualPlayer/TestUI/AnimatedVisualPlayerPage.xaml.cs

```diff
@@ -38,6 +38,7 @@ public FallbackGrid()
     /// <summary>
     /// An empty page that can be used on its own or navigated to within a Frame.
     /// </summary>
+    [AddToTestInventory(Name = "AnimatedVisualPlayer", Icon = "Animations.png")]
```

Comment:
Conventionally attributes are declaratively named. So this could be something like [TopLevelTestPage] or something like that.